ChatEventHandler.py

```diff
@@ -0,0 +1,117 @@
+'''
+Handles all the chat events and prints to the console in a readable format
+'''
+import config as configuration
+
+CONFIG = configuration
+
+
+def handler(data, chat):
```

Comment:
Generally it might be better to refactor this into an extensible class or set of classes. This works for fast script hacking, but I'd rather provide a solid foundation that others can build upon in our official example libraries.